src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -24,13 +30,27 @@ private static string GetFileNameFromResourceName(string s)
             return parts[parts.Length - 2] + "." + parts[parts.Length - 1];
         }
 
-        public int CreateEmptyProject()
+        public int CreateEmptyProject(Language language)
```

Comment:
I'm not 100% sure if I like the idea the idea of:
- having string input
- converting that to enum
- converting that to different string
- generating template from that different string.

Can we just use the original string instead and map it to directory name?